File: node/fizzbuzz.js
function fizzbuzz (n) {
  if ((n !== 0) && (n % 3 === 0) && (n % 5 !== 0)) {
    return 'Fizz'
  } else if ((n !== 0) && (n % 5 === 0) && (n % 3 !== 0)) {
    return 'Buzz'
  } else if ((n !== 0) && (n % 5 === 0) && (n % 3 === 0)) {
    return 'FizBuzz'
  }
  return n
}

module.exports = fizzbuzz

File: python/fizzbuzz.py
def fizzbuzz(n):
    if n != 0 and n % 3 == 0 and n % 5 != 0: 
       return 'Fizz'
    elif n != 0 and n % 5 == 0 and n % 3 != 0:
        return 'Buzz'
    elif n != 0 and n % 5 == 0 and n % 3 == 0:
        return 'FizzBuzz'
    return n
